plugins/github-deployments/src/components/GithubDeploymentsCard.tsx

```diff
@@ -67,6 +73,10 @@ export const GithubDeploymentsCard = ({
   columns?: TableColumn<GithubDeployment>[];
 }) => {
   const { entity } = useEntity();
+  const host: string | undefined = [
```

Comment:
const [host] = [entity?.metadata.annotations?.[SOURCE_LOCATION_ANNOTATION],
    entity?.metadata.annotations?.[LOCATION_ANNOTATION]].filter(Boolean);

Reply:
Didn't we make a helper for this somewhere 🤔

Reply:
🤷